components/invitation_modal/invitation_modal.jsx

```diff
@@ -93,21 +93,21 @@ export default class InvitationModal extends React.Component {
             this.setState({step: STEPS_INITIAL, hasChanges: false, lastInviteChannels: [], lastInviteMesssage: '', prevStep: this.state.step});
         }
         if (this.modal && this.modal.current) {
-            this.modal.current.resetFocus();
+            this.modal.current.enforceFocus();
```

Comment:
Gah, sorry about this @jespino! I didn't track that `resetFocus` had been used elsewhere when I removed it.

Reply:
No problem, is not a very common usage of methods inside components while using React, but sometimes is used.

Reply:
Does this not break TypeScript? `enforceFocus` is a private method on `FullScreenModal`